lib/view_component/test_helpers.rb

```diff
@@ -163,10 +163,20 @@ def with_controller_class(klass)
     # end
     # ```
     #
+    # To specify a request method, pass the method param:
+    #
+    # ```ruby
+    # with_request_url("/users/42", method: "POST") do
+    #   render_inline(MyComponent.new)
+    # end
+    # ```
+    #
     # @param path [String] The path to set for the current request.
     # @param host [String] The host to set for the current request.
-    def with_request_url(path, host: nil)
+    # @param method [String] The request method to set for the current request.
+    def with_request_url(path, host: nil, method: nil)
```

Comment:
maybe rename method to `http_method` for clarlrty

Reply:
It's indeed a clearer name but I'm a bit reluctant to rename the keyword since it would break people's code.

Reply:
I'm sorry it's my first review since I got back from vacation and I guess I'm not 100% ready to work yet 🫣
I did not realize this was a new option introduced by this PR. 😅
So yeah, I'm okay with renaming it if others think it's clearer too.

Reply:
@Spone @aduth considering your responses I feel sufficient context is provided